tests/Plugin/InvokeStrategy/FinderInvokeStrategyTest.php

```diff
@@ -42,7 +42,7 @@ protected function setUp()
         $this->actionEvent = new DefaultActionEvent(QueryBus::EVENT_INVOKE_FINDER, new QueryBus(), [
             QueryBus::EVENT_PARAM_MESSAGE => new CustomMessage('I am a query'),
             QueryBus::EVENT_PARAM_MESSAGE_NAME => CustomMessage::class,
-            QueryBus::EVENT_PARAM_DEFERRED => new Deferred(),
+            QueryBus::EVENT_PARAM_PROMISE => new Deferred(),
```

Comment:
@codeliner I believe a promise is now given not a deferred object. So maybe get promise from newly instaniated deferred object?